src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/Internal/TopicMessages.cs

```diff
@@ -42,9 +42,9 @@ internal sealed class Count
     }
 
     /// <summary>
-    /// TBD
+    /// Get all subscribers for a given topic.
     /// </summary>
-    internal sealed class CountSubscribers
+    public sealed class CountSubscribers
```

Comment:
Just made this `internal` class `public` - left it inside the `.Internal` namespace though (which I think this fine - it's public but we reserve the right to change it, per our API guidelines: https://getakka.net/community/contributing/api-changes-compatibility.html#akkanet-api-versioning-policy )

Reply:
Might want to decorate this class with `[ApiMayChange]` attribute for documentation purposes

Reply:
Will do